src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -362,7 +376,8 @@ public GoogleCredentials getPetSACredentials() {
 
   /** Return true if the user credentials are expired or do not exist on disk. */
   public boolean requiresReauthentication() {
-    if (userCredentials == null) {
+    checkForAppDefaultCredentials();
```

Comment:
nit: I think this method should just check if existing credentials are expired, and not pull them, and we can keep the logic about which creds to use in the login method.

Reply:
loadExistingCredentials is called by `BaseCommand` when every command is run. But `loadExistingCredentials` will set `googleCredentials` to null when there is no user credentials. I was not able to get ADC in that method for some reason (not sure if it has something to do with it is called by the `BaseCommand`.